app/assets/stylesheets/widgets.scss

```diff
@@ -282,15 +282,7 @@
 // Sidebar sponsors
 .crayons-sponsorship-article {
   margin-bottom: 10px;
-
-  img {
```

Comment:
This style made sense until we added the `text-styles` class which squared away image styling in general for this area.

Reply:
Looks much better this way 🙌🏼